INSTALL.md

```diff
@@ -1,6 +1,9 @@
 - Install ACADO for your system by following the instructions at
-  http://acado.github.io/. This package assumes that the header files get placed in
+  http://acado.github.io/.
+  * This package assumes that the header files get placed in
   `/usr/local/include/acado`, edit `setup.py` with the correct paths if this is
   not the case.
 - Install the Python requirements found in `requirements.txt` with `pip` or otherwise.
+  * Both `eigency` and `acadopy` require GCC to be used, so you may need to
```

Comment:
`eigency` is not really used. I used it as an attempt to expose the internal arrays used by Acado. I still think we'll have to do that at some point. Not sure `eigency` is the right library.